packages/govuk-frontend/src/govuk/components/task-list/_index.scss

```diff
@@ -0,0 +1,72 @@
+@include govuk-exports("govuk/component/task-list") {
+  $govuk-task-list-focus-colour: govuk-colour("black");
+  $govuk-task-list-focus-background-colour: govuk-colour("yellow");
+  $govuk-task-list-hover-colour: govuk-colour("light-grey");
+
+  .govuk-task-list {
+    @include govuk-font($size: 19);
+    margin-top: 0;
+    @include govuk-responsive-margin(6, "bottom");
+    padding: 0;
+    list-style-type: none;
+    @include govuk-media-query($from: tablet) {
+      min-width: 550px;
+    }
```

Comment:
This can force the task list to overflow it's container, which can cause issues if for example it's used in grid column where there is anything else in the same row.

![Screenshot 2023-06-08 at 11 03 53](https://github.com/alphagov/govuk-frontend/assets/121939/8573dc93-c53e-412b-bcb0-116ddf74b877)

Reply:
I'd also suggest adding a full-page example to the review app that shows how the task list works in the context of a page, which would make it easier to catch things like this.

Reply:
Opps! 😳 

Good idea about adding a full page example to the review app!